src/components/side_panel/conditional_formatting/conditional_formatting.xml

```diff
@@ -101,10 +101,10 @@
               <t t-set="rule" t-value="state.rules.iconSet"/>
             </t>
             <div class="d-flex justify-content-end p-4">
-              <button t-on-click="switchToList" class="o-sidePanelButton o-cf-cancel">
+              <button t-on-click="switchToList" class="btn btn-secondary me-2 o-cf-cancel">
                 Cancel
               </button>
-              <button t-on-click="saveConditionalFormat" class="o-sidePanelButton o-cf-save">
+              <button t-on-click="saveConditionalFormat" class="btn btn-primary o-cf-save">
```

Comment:
this will definitely change the look of the button. not that it's a bad thing though

Reply:
Yes, and a good point is that we will not have to change the colours in Odoo, it's automatic